packages/lib/passport-interface/test/MockFeedApi.ts

```diff
@@ -25,10 +26,19 @@ class MockFeedError extends Error {
   }
 }
 
+async function extractPayload(
+  credential: SerializedPCD<SemaphoreSignaturePCD>
```

Comment:
```suggestion
  credential: Credential
```

i suggest doing a repo-wide search for `SerializedPCD<SemaphoreSignaturePCD>` to see if there's any other instances of this simplification that you missed

Reply:
I thought I had done this! Must have narrowed the search incorrectly.